### DISCUSSION TITLE
Packages that are not being maintained should be left open to other developers

### DISCUSSION THREAD

**Discussion by ccencisoj** [npm/feedback#700] [Registry] [answered]
There are many packages that have few downloads and more than a year since their release. At the beginning, the developers have the vision that these packages will contribute something to the community, they maintain them during the first months and are attentive to the problems that could arise. After seeing that there are no developers interested in using their packages for "x" reason, they lose motivation and abandon them.

This week I developed a tool that makes it easy for developers to load files from local storage, once I was done I thought to publish it to npmjs with the name “react-storage”, but the name was already taken. In npmjs, “react-storage” is a package that was published by “yuanyan”, it has about five weekly downloads (bots) and its last update was seven years ago, from this description it can be concluded that this package is abandoned .

I propose that leave free packages that are six months old since their last publication and less than ten weekly downloads. This is so that other developers have the opportunity to publish their packages with the name of the packages that have been abandoned.

**Comment by monishcm** [comment, marked as answer]
Thanks for your feedback. npm being a public registry, we do not remove abandoned packages for historical value. If you wish to maintain an abandoned project or willing to take over the name please write to our support. Please note that we have over 3 months of backlog for name disputes and reviewing yours might take some time. Alternatively, you can scope your package to retain the name.
